src/app/core/services/label.service.ts

```diff
@@ -10,17 +10,42 @@ import { User } from '../models/user.model';
   providedIn: 'root'
 })
 export class LabelService {
-  private severityLabels: Label[];
-  private typeLabels: Label[];
-  private responseLabels: Label[];
-  private labelColorMap: Map<string, string>;
-
+  private severityLabels: Label[] = [];
+  private typeLabels: Label[] = [];
+  private responseLabels: Label[] = [];
+  private statusLabels: Label[] = [];
+  private labelArrays = {
+    severity: this.severityLabels,
+    type: this.typeLabels,
+    response: this.responseLabels,
+    status: this.statusLabels
+  };
+  private labelColorMap: Map<string, string> = new Map();
+
+  // Preset Labels.
```

Comment:
Redundant comment (also, in the very next line, the variable is named 'required labels', not 'preset labels'). Either give a more detailed comment (preferred) or remove.

Reply:
Sure, I'll remove the comment and standardise the use of `Required`.